frontend/src/components/OrganizationList/OrganizationList.tsx

```diff
@@ -56,19 +68,23 @@ export const OrganizationList: React.FC<{
         spacing={2}
         style={{ margin: '0 auto', marginTop: '1rem', maxWidth: '1000px' }}
       >
-        <Grid item>
-          <Paper
-            elevation={0}
-            classes={{ root: classes.cardRoot }}
-            style={{ border: '1px dashed #C9C9C9', textAlign: 'center' }}
-            onClick={() => setDialogOpen(true)}
-          >
-            <h1>Create New {parent ? 'Team' : 'Organization'}</h1>
-            <p>
-              <Add></Add>
-            </p>
-          </Paper>
-        </Grid>
+        {userLevel === GLOBAL_ADMIN ?(
```

Comment:
We can just do this instead of using a ternary operator:
```suggestion
        {userLevel === GLOBAL_ADMIN && (
```

(remove the `:""` at the end)

Reply:
Oh right got it.